app/Http/Controllers/PartyController.php

```diff
@@ -777,7 +777,7 @@ public function notifyHostsOfRsvp($user_event, $event_id)
             $hosts = null;
         }
 
-        if (! is_null($hosts)) {
+        if (count($hosts)) {
```

Comment:
I'm not sure how Laravel in a production environment if this is null - is it OK?  

Recent PHPs will give a warning if reporting is E_ALL.  https://3v4l.org/bZCgq
Laravel doesn't like warnings in production.

Reply:
Thanks - will make it robust against that.